kythe/go/extractors/gcp/README.md

```diff
@@ -0,0 +1,31 @@
+# Kythe Extracting on GCP
+
+This package contains nothing of note, but will eventually support extracting
+Kythe Compilation Units on Google Cloud Platform.
+
+## Cloud Build
+
+Documentation for Cloud Build itself is available at
+https://cloud.google.com/cloud-build/.
+
+For the rest of this test documentation, we'll assume you've run those setup
+instructions.  Additionally, you should make an environment variable for your
+gs bucket:
+
+```
+export BUCKET_NAME="your-bucket-name"
+```
+
+## Hello World Test
+
+To make sure you have done setup correctly, we have a shim binary at
```

Comment:
shim (n): something used to align parts, make them fit, or reduce wear.

"toy", "example", etc would be a better word.

Reply:
Yeaaa probably.

Reply:
you don't pop up in my list of reviewers.